var/spack/repos/builtin/packages/vecgeom/package.py

```diff
@@ -74,6 +74,7 @@ class Vecgeom(CMakePackage, CudaPackage):
         deprecated=True,
     )
 
+    patch("remove-template-kw.patch", when="@:1.2.10%clang")
```

Comment:
Can you see if the following works for you:
```suggestion
    patch("https://gitlab.cern.ch/VecGeom/VecGeom/-/merge_requests/1251.patch", when="@:1.2.10%clang", sha256="72501ffa9a8c6136fbcbfa83a404fdd7ad4dbd144de06a937c5ae5456af60c0c")
```
Then you won't need to include the manual patch file.